src/PostgREST/QueryBuilder.hs

```diff
@@ -433,12 +433,8 @@ pgFmtFilter table (Filter fld (OpExpr hasNot oper)) = notOp <> " " <> case oper
       Just True  -> emptyValForIn
       Nothing    -> emptyValForIn
 
-   Fts mode lang val ->
-     pgFmtFieldOp "fts" <> " " <> case mode of
-       Normal -> "to_tsquery("
-       Plain  -> "plainto_tsquery("
-       Phrase -> "phraseto_tsquery("
-     <> maybe "" (flip (<>) ", " . pgFmtLit) lang <> unknownLiteral val <> ") "
+   Fts op lang val ->
+     pgFmtFieldOp op <> "(" <> maybe "" (flip (<>) ", " . pgFmtLit) lang <> unknownLiteral val <> ") "
```

Comment:
How about a little whitespace and a section rather than flip?

```haskell
pgFmtFieldOp op <> "("
  <> maybe "" ((<> ", ") . pgFmtLit) lang
  <> unknownLiteral val <> ") "
```